great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -882,6 +886,70 @@ def column_reflection_fallback(self):
             ]
         return col_dict
 
+    ###
+    ###
+    ###
+    #
+    # Column Aggregate Expectation Implementations
```

Comment:
I'm confused. expect_table_row_count_to_equal_other_table isn't a Column Aggregate Expectation, right?  It's just a generic expectation

Reply:
Correct -- fixing!

Reply:
Updated